app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -1158,6 +1160,7 @@ private void checkGzipSupport(Response r) {
 
 
     private static final String LAST_NIGHTSCOUT_BATTERY_LEVEL = "last-nightscout-battery-level";
+    private static final String LAST_G5TX_BATTERY_LEVEL = "last-g5tx-battery-level";
```

Comment:
This constant indicates that it is only valid for G5. Is there a generalized constant defined in Nightscout?
Even if not, we should name use a generalized name like DEXNT for new technology or simply DEX because the battery information was not available with G4.